content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -15,7 +15,6 @@ Network plugins in Kubernetes come in a few flavors:
 
 * CNI plugins: adhere to the [Container Network Interface](https://github.com/containernetworking/cni) (CNI) specification, designed for interoperability.
   * Kubernetes follows the [v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) release of the CNI specification.
-* Kubenet plugin: implements basic `cbr0` using the `bridge` and `host-local` CNI plugins
```

Comment:
It sounds like the _only_ game in town is now CNI plugins. If that's true (I think it is), we need to rewrite the page overview to make that clear.

Reply:
Thanks @sftim, In https://github.com/kubernetes/kubernetes/pull/106907, `--network-plugin` was removed. Can we remove this file?

Reply:
Which file do you propose to remove @mk46?

Reply:
`content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md`